sympy/geometry/polygon.py

```diff
@@ -374,6 +374,63 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+
+    def second_moment_of_area(self, p=None):
+        """Returns the second moments and products of area of a two dimensional polygon.
+
+        Parameters
+        ==========
+
+        p : Point, two-tuple of sympifiable objects, or None
+            p is the point about which second moment of area is to be found.
+            If None, the point will be taken as the centroid of the polygon.
+
+        Notes
+        =====
+
+        Returns the second moment of area and products of area of a two dimensional polygon
+        about the axis passing through the point p. If p is None then about the axis passing
```

Comment:
Keep the comment regarding `if p is None ... yada yada` here under `Notes` heading.